CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+
+## Adding flags
+
+Flags are stored in the `flags` folder. To add a new one simply:
+
+1. Create a new js file with the id of the flag `flagid.js`
+2. Fill the flag out with it's info.
+3. Add the flag to the `flags/index.js` file.
+4. Add the flag svg file to `flags/svg` folder. Generally I source the SVGs from wikipedia, if you need any help creating them feel free to message me on [Twitter](https://twitter.com/joehart).
+5. Then import the svg to the `flagid.js` file.
+6. Raise a PR for adding the flag
+
+Note if adding multiple flags, please raise a PR per individual flag added.
+
```

Comment:
Something to consider: It could be nice to add a sub-section about how to run the site locally. That way, people know how to check if their flag SVG is rendering correctly under the `/flags` route before opening their PR.